src/lib/sidebar/panelTests.ts

```diff
@@ -0,0 +1,270 @@
+    utils.setCWD();
+    const templateHandle = templateUtils.getTemplateHandle();
+    const templateType = await templateUtils.getTemplateType();
+    const testNames = (await this.liquidTestRunner.listTestNames()) || [];
+    const gridLayout = `grid-template-columns="2fr 1fr"`;
+    const disabledLabel = this.clickableButtons() ? "" : "disabled";
+    const disabledReverseLabel = this.clickableButtons() ? "disabled" : "";
+    const liquidTestsRows = testNames.map((testName: string) => {
+      return /*html*/ `<vscode-data-grid-row grid-columns="2">
+              <vscode-data-grid-cell grid-column="1">
+                ${testName}
+              </vscode-data-grid-cell>
+              <vscode-data-grid-cell grid-column="2" class="vs-actions">
+                <vscode-button appearance="icon" aria-label="Run-test" class="run-test" title="Run liquid test" data-html-type="none" data-test-name="${testName}" data-template-type="${templateType}" data-template-handle="${templateHandle}" ${disabledLabel}>
+                  <i class="codicon codicon-debug-alt"></i>
+                </vscode-button>
+                <vscode-button appearance="icon" aria-label="generate-html-input" class="run-test" title="Generate HTML (Input)" data-html-type="input" data-test-name="${testName}" data-template-type="${templateType}" data-template-handle="${templateHandle}" ${disabledLabel}>
+                  <i class="codicon codicon-file-binary"></i>
+                </vscode-button>
+                <vscode-button appearance="icon" aria-label="generate-html-output" class="run-test" title="Generate HTML (Preview)" data-html-type="preview" data-test-name="${testName}" data-template-type="${templateType}" data-template-handle="${templateHandle}" ${disabledLabel}>
+                  <i class="codicon codicon-file-pdf"></i>
+                </vscode-button>
+              </vscode-data-grid-cell>
+            </vscode-data-grid-row>`;
+    });
+
+    const allTestsRow =
+      /*html*/
+      `<vscode-data-grid-row grid-columns="2">
+            <vscode-data-grid-cell grid-column="1">
+              All tests
+            </vscode-data-grid-cell>
+            <vscode-data-grid-cell grid-column="2" class="vs-actions">
+              <vscode-button appearance="icon" aria-label="Run-all-test" class="run-test" title="Run all liquid test" data-html-type="none" data-test-name="" data-template-type="${templateType}" data-template-handle="${templateHandle}" ${disabledLabel}>
+                <i class="codicon codicon-debug-alt"></i>
+              </vscode-button>
+              <vscode-button appearance="icon" aria-label="generate-html-input" disabled>
+                <i class="codicon codicon-file-binary"></i>
+              </vscode-button>
+              <vscode-button appearance="icon" aria-label="generate-html-output" disabled>
+                <i class="codicon codeicon-file-pdf"></i>
+              </vscode-button>
```

Comment:
```suggestion
```

I don't think the render is supported for the "All tests" option?

Reply:
I think it's a good to keep them as always disabeled. For two reasons:
- You explicitly now it's not possible.
- More important: icons layout, it just looks nicer since all icons are aligned